templates/web.ssl.template.yml

```diff
@@ -0,0 +1,38 @@
+run:
+  - exec:
+     cmd:
+       - "mkdir -p /shared/ssl/"
+       - "[ -e /shared/ssl/dhparams.pem ] || openssl dhparam -out /shared/ssl/dhparams.pem 2048"
+  - replace:
+     filename: "/etc/nginx/conf.d/intercity.conf"
+     from: /server.+{/
+     to: |
+       server {
+         listen 80;
+         rewrite ^ https://$$ENV_HOSTNAME$request_uri? permanent;
```

Comment:
haha no, this is some magic in Pups (The system I use to parse these yml files inside the docker container: https://github.com/SamSaffron/pups/)